.gitignore

```diff
@@ -20,3 +20,9 @@ models/px4flow/px4flow.sdf
 models/r200/r200.sdf
 models/sf10a/sf10a.sdf
 models/standard_vtol/standard_vtol.sdf
+models/temp*
+worlds/temp*
+models/cupcar/cupcar.sdf
+models/r1_rover/r1_rover.sdf
+models/rover/rover.sdf
+models/tiltrotor/tiltrotor.sdf
```

Comment:
This is already upstream: https://github.com/PX4/PX4-SITL_gazebo/blob/master/.gitignore#L25.